android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -107,6 +109,8 @@ class RNPlayerView(val context: ReactApplicationContext) :
         getViewTreeObserver().addOnGlobalLayoutListener { requestLayout() }
     }
 
+    var pictureInPictureConfig: PictureInPictureConfig? = null
```

Comment:
I think we should move this to where `pictureInPictureHandler` lives.

Reply:
fixed in 90b13a1